src/colour/social/index.md

```diff
@@ -57,53 +59,55 @@ Our social palette has:
 
 ## Use colour to reflect tone
 
-Our updated palette has been developed to allow a range in expression across the inform to inspire scale. There are moments where the brand needs to feel functional and serious, guiding users seamlessly to the content and services they need. With the introduction of new channels such as social, there are also moments where the brand needs impact and visual differentiation.
+Our updated palette has been developed to allow a range in expression across the inform to inspire scale. 
+
+There are moments where the brand needs to feel functional and serious, guiding users seamlessly to the content and services they need. With the introduction of new channels such as social, there are also moments where the brand needs impact and visual differentiation.
 
-Within our palette there are two approaches to colour application; tonal colours and companion colours.
+Within our palette there are two approaches to colour application: tonal colours and companion colours.
 
 Depending on tonal requirement, each can be used to achieve a different level of expression. The following guidance details the use of these two approaches.
 
 <!-- Obviously, that's not the right heading, needs proper design -->
 
 ### Inform <--------> Inspire
 
-![TODO](./tone-black-and-white.png)
+![Title graphic for "National mourning guidance" in black and white.](./tone-black-and-white.png)
```

Comment:
(Optional) I've got the feeling I've asked this before (maybe a month or so ago), but what is a "title graphic"? It seems to be a very vague term (if I understand it correctly). Although maybe it's intentionally vague?
Same for the following mentions of the term.

Reply:
Trying to be more descriptive than just "graphic", and get across that it's a graphic of some text, and that it looks like a "title card", which is a common term for these in film and video.

Would "title card graphic" be clearer? Or just say "graphic" and keep it minimal?

Reply:
I don't think "image of text" would be good in this case. It is too broad.

I think we should involve the rest of the squad. Maybe a Slack thread to get opinions of the others?